<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgBarAndIcon</class>
 <widget class="QMainWindow" name="ProgBarAndIcon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Taskbar progress bar and overlay icon demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="textVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Progress bar status:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="cbProgressType">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>No progress</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Indeterminate</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Normal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Error</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Paused</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Overlay Icon:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cbOverlayIcon">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Paused</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/pause.png</normaloff>:/pause.png</iconset>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Playing</string>
           </property>
           <property name="icon">
            <iconset resource="icons.qrc">
             <normaloff>:/play.png</normaloff>:/play.png</iconset>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
